demos/client-bundle-example/src/index.html

```diff
@@ -22,6 +22,7 @@
                 <div
                     class="procaptcha"
                     data-theme="light"
+                    data-captcha-type="image"
```

Comment:
still set to image captcha?

Reply:
Yes, for a follow up PR though to keep size reasonable. Added to this sprint

https://github.com/orgs/prosopo/projects/4/views/1?pane=issue&itemId=54753652